.gitignore

```diff
@@ -28,3 +28,6 @@
 
 # Ignore application configuration
 /config/application.yml
+
+# Ignore environment configurations
+.env.*
```

Comment:
it may be a good idea to add a `.env.example` file so it's easier to configure it